sdk/compute/Microsoft.Azure.Management.Compute/src/Customizations/GalleryApplicationVersionUpdate.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.Azure.Management.Compute.Models
+{
+    public partial class GalleryApplicationVersionUpdate
+    {
+        GalleryApplicationVersionUpdate(
+            GalleryApplicationVersionPublishingProfile publishingProfile,
+            string id,
+            string name,
+            string type,
+            IDictionary<string, string> tags,
+            string provisioningState,
+            ReplicationStatus replicationStatus)
```

Comment:
As per our wiki: https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/182581/Creating-Customization-Constructor?anchor=how-to-create-custom-constructors
The parameters after the parameter directly after the new parameter need to be optional in the customization.
So in this case replicationStatus needs to be optional have a default value. Similar to this: https://github.com/Azure/azure-sdk-for-net/blob/feature/gallery-2022-03-03/sdk/compute/Microsoft.Azure.Management.Compute/src/Customizations/GalleryApplicationVersionPublishingProfile.cs#L15